src/status_im2/contexts/wallet/edit_account/view.cljs

```diff
@@ -15,9 +15,10 @@
 (defn- show-save-account-toast
   [updated-key theme]
   (let [message (case updated-key
-                  :name  :t/edit-wallet-account-name-updated-message
-                  :color :t/edit-wallet-account-colour-updated-message
-                  :emoji :t/edit-wallet-account-emoji-updated-message
+                  :name                     :t/edit-wallet-account-name-updated-message
+                  :color                    :t/edit-wallet-account-colour-updated-message
+                  :emoji                    :t/edit-wallet-account-emoji-updated-message
+                  :prod-preferred-chain-ids :t/edit-wallet-network-preferences-updated-message
```

Comment:
can we change this from `:prod-preferred-chain-ids` to `:network-configuration` as :prod-preferred-chain-ids is more implementation details

Reply:
@J-Son89 
In this implementation `:prod-preferred-chain-ids` should corresponds updating-key

Reply:
but this is just for the toast message?

Reply:
ah I see now okay, let's leave it as is so 👍